.github/workflows/generate_baseline_profile.yml

```diff
@@ -43,6 +47,12 @@ jobs:
           path: ./android-sdk-benchmark
           token: ${{ secrets.GH_ANDROID_SDK_TOKEN || secrets.token }} # NOTE: read android-sdk-benchmark
 
+      - name: Update gradle.properties with the proper SDK version
+        working-directory: android-sdk-benchmark
+        run: |
+          sed -i "s/swazzler_version=.*/swazzler_version=${{ env.ANDROID_SDK_VERSION }}/" gradle.properties
```

Comment:
As an FYI you can supply this as a [command-line arg to Gradle](https://docs.gradle.org/current/userguide/build_environment.html#priority_for_configurations) if you want a solution that doesn't create a dirty working tree

Reply:
that's a great idea I'll use that 🙌